leetduel-backend/src/main.py

```diff
@@ -82,7 +110,16 @@ async def create_party(sid: str, data: dict) -> None:
     party_code = generate_party_code()
     parties[party_code] = {
         "host": sid,
-        "players": [{"sid": sid, "username": data["username"], "passed": False}],
+        "players": [
+            {
+                "sid": sid,
+                "username": data["username"],
+                "passed": False,
+                "current_score": 0,
+                "total_score": 0,
+                "finish_order": None,
+            }
+        ],
```

Comment:
@jeffreykim nit: longer term wanna make this players thing a dataclass for readability